modules/custom/openy_repeat/templates/openy-repeat-schedule-dashboard--table.html.twig

```diff
@@ -154,7 +154,7 @@
         </div>
 
         <div class="schedule-dashboard__modal--footer">
-          <a class="btn btn-lg btn-primary" v-bind:href="'node/' + locationPopup.nid">{{ 'View branch'|t }}</a>
+          <a class="btn btn-lg btn-primary" v-bind:href="'{{ url('<front>') }}node/' + locationPopup.nid">{{ 'View branch'|t }}</a>
```

Comment:
I do not like the idea of using URL in the format /node/nid but it works since there is a redirect.